site/collections/_projects/sustainability-leadership-kosova.md

```diff
@@ -5,8 +5,33 @@ subtitle: Used data to influence an ongoing governmental draft on deforestation
 
-Faruk has used satellite imagery using a GIS model for his analysis and has even made a predictive analysis for future deforestation. The aim of this project was to use this data to influence an ongoing governmental draft on deforestation regulation strategy as well as empower Civil Society Organisations, NGOs, and activists to increase pressure on the government to act.
+One of SLK’s significant challenges has been addressing deforestation in Kosovo. With the assistance of local GIS expert Faruk Foniqi, SLK identified the rate of deforestation, including illegal activities, using satellite imagery and GIS analysis. This data aimed to influence governmental draft regulations on deforestation and empower Civil Society Organisations, NGOs, and activists to pressure the government for action.
 
-There is a now a new blogpost with more details. Find out more about this project [here]({% post_url 2023-06-22-sustainability-leadership-kosova %})
+Despite having the story and the initial data, SLK faced a challenge in effectively presenting this information to raise awareness. This highlights the ongoing need for innovative communication strategies to effectively mobilize support and drive impactful environmental action.
+
+# The Solution
+With the guidance and insights from the team of SLK, the expertise in storytelling of Ashraf Ramzy from MasterStory, and the dedication of CorrelAid volunteers, the combined efforts were focused on transforming the narrative around deforestation in Kosovo. Our volunteers combined data analysis with innovative data visualization techniques to create a compelling and persuasive story. This comprehensive approach turned raw data into meaningful insights, designed to captivate audiences and apply strategic pressure on the government’s deforestation plans. Through this collaboration, SLK, CorrelAidNL, and MasterStory volunteers worked together to breathe life into the data, making it relevant and impactful.
+
+![kosova](/images/kosova.png){: width="1200" height="900"}
+
+# The Impact
+The impact was significant: the campaign successfully raised public awareness and compelled governmental action to address deforestation in Kosovo. By presenting data in an accessible and visually engaging manner, our volunteers enabled a broader audience to understand the urgency of the deforestation crisis and inspired them to take action. The #MoseMerrMalin campaign, born from this collaboration, mobilized citizens, businesses, and the government to protect and restore Kosovo's forests. This initiative aims to grow the economy, secure jobs, ensure clean air, and foster a thriving natural environment, encouraging collective responsibility and decisive action.
```

Comment:
It raised public awareness, but do we have any information on the result of this campaign? Did anything change?